publish.sh

```diff
@@ -62,3 +62,27 @@ git add .
 git status
 git commit -m "Published by Buildkite $BUILDKITE_BUILD_URL"
 git push origin "$GITHUB_PAGES_BRANCH"
+
+
+echo ">> verifying chart version update"
+helm repo add komodorio https://helm-charts.komodor.io
+helm repo update
+helm repo list
+
+for x in {1..10}; do
```

Comment:
I think we might need to do `helm repo update` again in the loop no? @nirbenator